specification/src/chapter2.adoc

```diff
@@ -603,20 +691,19 @@ Verification ensures the system has loaded authorized software
 | Requirement
 
 | CAT_NNN
-| A system MUST only use authorizations from trusted signers.
+| A system MUST only use authorizations from trusted authority.
 |===
 
-* Direct verification requires a signed image autorization from a trusted
-signer +
-For example, a signed image header, or a separately signed authorization
+* Direct verification requires a signed image authorization from a trusted
```

Comment:
```suggestion
* Direct verification requires a signed image from a trusted
```

Reply:
The terms "authorized software" "measured software" and "verified software" are defined in the ecosystem chapter. I don't think we need to introduce new terms here.

Examples of authorizations in this case are in the sentence immediately following.